docs/async_dispatch.rst

```diff
@@ -10,6 +10,8 @@ program:
 >>> import jax.numpy as jnp
 >>> from jax import random
 >>> x = random.uniform(random.PRNGKey(0), (1000, 1000))
+>>> # Printing the result or repr(result) or str(result) will block until the
+>>> # value is ready. That's why the value is present in the output below.
```

Comment:
```suggestion
>>> # value is ready.
```

I'm not sure but I suspect this second sentence isn't clear without more context.

Reply:
Done